pkg/initialize/run.go

```diff
@@ -99,15 +106,10 @@ func DownloadPolicies() error {
 		return fmt.Errorf("failed to checkout git branch '%v'. error: '%v'", branch, err)
 	}
 
-	// cleaning the existing cached policies at basePath
-	if err = os.RemoveAll(basePath); err != nil {
-		return fmt.Errorf("failed to clean up the directory '%s'. error: '%v'", basePath, err)
-	}
-
-	// move the freshly cloned repo from tempPath to basePath
-	if err = os.Rename(tempPath, basePath); err != nil {
-		return fmt.Errorf("failed to install policies to '%s'. error: '%v'", basePath, err)
-	}
-
 	return nil
 }
+
+func connected(url string) bool {
+	_, err := http.Get(url)
+	return err == nil
```

Comment:
Thinking about the use case for when it can't get `url` - do we want to print the error detail? User isn't sure if it's can't connect, can't lookup dns name, or something else...

Reply:
we can tackle that in #576